config/google_cloud_exporter/prometheus/README.md

```diff
@@ -6,23 +6,95 @@ The Prometheus Receiver can be used to send prometheus style metrics scraped fro
 
 See the [prerequisites](../README.md) doc for Google Cloud prerequisites.
 
-## Configuration
+Ensure your applications support prometheus metrics, or have prometheus exporters configured.
 
-An example configuration is located [here](./config.yaml).
+## Usage
+
+Modify the example configuration's `prometheus` receiver. Update
+the `job_name` value to reflect your environment. Also update the
+`targets` value(s) to point to your application's prometheus exporter(s).
+
+For example, if you have five Nginx systems that need to be
+monitored, your receiver configuration would look like this:
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'nginx'
+        scrape_interval: 60s
+        static_configs:
+        - targets:
+          - 'nginx-0:9113'
+          - 'nginx-1:9113'
+          - 'nginx-2:9113'
+          - 'nginx-3:9113'
+          - 'nginx-4:9113'
+```
+
+You can have multiple scrape configs, for targeting multiple applications.
+
+```yaml
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+      - job_name: 'nginx'
+        scrape_interval: 60s
+        static_configs:
+        - targets:
+          - 'nginx-0:9113'
+      - job_name: 'redis'
+        scrape_interval: 60s
+        static_configs:
+        - targets:
+          - 'redis-0:9900'
+```
+
+### Deploy
 
 1. Copy [config.yaml](./config.yaml) to `/opt/observiq-otel-collector/config.yaml`
-2. Restart the collector: `sudo systemctl restart observiq-otel-collector`
+2. Modify the configuration to reflect your environment (See [Usage](./README.md#usage))
+3. Restart the collector: `sudo systemctl restart observiq-otel-collector`
```

Comment:
Do we assume linux for all our config examples?

Reply:
I think so, we probably could include Windows at some point. The config path and restart commands would need to be changed or made "generic".